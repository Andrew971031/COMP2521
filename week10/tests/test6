900089
900037
900061
900103
900007
900019
900149
900139
900091
900121
900143
900157
900187
900217
900161
900169
900233
900253
900259
900283
900293
900307
900331
900241
900329
900443
900287
900349
900409
900461
900481
900563
900569
900511
900491
900539
900551
900553
900577
900583
900589
900587
900397
900649
900659
900607
900673
900593
900623
900689
900773
900701
900751
900737
900671
900719
900803
900763
900743
900817
900761
900797
