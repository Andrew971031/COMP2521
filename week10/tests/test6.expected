Tree:
             900089
               / \
              /   \
             /     \
            /       \
           /         \
          /           \
         /             \
        /               \
     900037           900103
      / \               / \
     /   \             /   \
    /     \           /     \
   /       \         /       \
900007   900061   900091   900149
   \                         / \
 900019                     /   \
                           /     \
                          /       \
                         /         \
                      900139     900157
                       / \           \
                      /   \        900187
                     /     \         / \
                    /       \       /   \
                 900121   900143   /     \
                                  /       \
                               900161   900217
                                  \         \
                                900169    900233
                                              \
                                            900253
                                              / \
                                             /   \
                                            /     \
                                           /       \
                                        900241   900259
                                                     \
                                                   900283
                                                       \
                                                     900293
                                                       / \
                                                      /   \
                                                     /     \
                                                    /       \
                                                 900287   900307
                                                              \
                                                            900331
                                                              / \
                                                             /   \
                                                            /     \
                                                           /       \
                                                        900329   900443
                                                                   / \
                                                                  /   \
                                                                 /     \
                                                                /       \
                                                             900349   900461
                                                                \         \
                                                              900409    900481
                                                                /           \
                                                             900397       900563
                                                                            / \
                                                                           /   \
                                                                          /     \
                                                                         /       \
                                                                        /         \
                                                                     900511     900569
                                                                      / \           \
                                                                     /   \        900577
                                                                    /     \           \
                                                                   /       \        900583
                                                                900491   900539         \
                                                                             \        900589
                                                                           900551       / \
                                                                               \       /   \
                                                                             900553   /     \
                                                                                     /       \
                                                                                  900587   900649
                                                                                             / \
                                                                                            /   \
                                                                                           /     \
                                                                                          /       \
                                                                                         /         \
                                                                                      900607     900659
                                                                                       / \           \
                                                                                      /   \        900673
                                                                                     /     \         / \
                                                                                    /       \       /   \
                                                                                 900593   900623   /     \
                                                                                                  /       \
                                                                                               900671   900689
                                                                                                            \
                                                                                                          900773
                                                                                                            / \
                                                                                                           /   \
                                                                                                          /     \
                                                                                                         /       \
                                                                                                        /         \
                                                                                                       /           \
                                                                                                      /             \
                                                                                                   900701         900803
                                                                                                      \             / \
                                                                                                    900751         /   \
                                                                                                      / \         /     \
                                                                                                     /   \       /       \
                                                                                                    /     \   900797   900817
                                                                                                   /       \
                                                                                                900737   900763
                                                                                                 / \       /
                                                                                                /   \   900761
                                                                                               /     \
                                                                                              /       \
                                                                                           900719   900743

#nodes = 62,  #leaves = 21
Infix  : 900007 900019 900037 900061 900089 900091 900103 900121 900139 900143 900149 900157 900161 900169 900187 900217 900233 900241 900253 900259 900283 900287 900293 900307 900329 900331 900349 900397 900409 900443 900461 900481 900491 900511 900539 900551 900553 900563 900569 900577 900583 900587 900589 900593 900607 900623 900649 900659 900671 900673 900689 900701 900719 900737 900743 900751 900761 900763 900773 900797 900803 900817 
Prefix : 900089 900037 900007 900019 900061 900103 900091 900149 900139 900121 900143 900157 900187 900161 900169 900217 900233 900253 900241 900259 900283 900293 900287 900307 900331 900329 900443 900349 900409 900397 900461 900481 900563 900511 900491 900539 900551 900553 900569 900577 900583 900589 900587 900649 900607 900593 900623 900659 900673 900671 900689 900773 900701 900751 900737 900719 900743 900763 900761 900803 900797 900817 
Postfix: 900019 900007 900061 900037 900091 900121 900143 900139 900169 900161 900241 900287 900329 900397 900409 900349 900491 900553 900551 900539 900511 900587 900593 900623 900607 900671 900719 900743 900737 900761 900763 900751 900701 900797 900817 900803 900773 900689 900673 900659 900649 900589 900583 900577 900569 900563 900481 900461 900443 900331 900307 900293 900283 900259 900253 900233 900217 900187 900157 900149 900103 900089 
ByLevel: 900089 900037 900103 900007 900061 900091 900149 900019 900139 900157 900121 900143 900187 900161 900217 900169 900233 900253 900241 900259 900283 900293 900287 900307 900331 900329 900443 900349 900461 900409 900481 900397 900563 900511 900569 900491 900539 900577 900551 900583 900553 900589 900587 900649 900607 900659 900593 900623 900673 900671 900689 900773 900701 900803 900751 900797 900817 900737 900763 900719 900743 900761 
